ceph/ceph.py

```diff
@@ -525,9 +525,11 @@ def get_osd_metadata(self, osd_id, client=None):
                 return metadata
         return None
 
-    def osd_check(self, client):
-
-        out, err = client.exec_command(cmd='sudo ceph -s -f json')
+    def osd_check(self, client, cluster_name=None):
```

Comment:
[1]Have default argument value(cluster_name='ceph') rather than `None`. since 'ceph' is used as default cluster name.